.github/workflows/check-project-assignment.yml

```diff
@@ -0,0 +1,61 @@
+name: "[Check] Project/Issue Assignment"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - ready_for_review
+      - labeled
+      - synchronize
+
+jobs:
+  check-project-or-issue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Check if PR is assigned to a project or an issue
+        env:
+          GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
+        run: |
+          # Fetch PR labels
+          PR_LABELS=$(gh pr view ${{ github.event.pull_request.number }} --json labels --jq '.labels[].name')
+
+          # Check for "no-project" label
+          if echo "$PR_LABELS" | grep -q "^no-project$"; then
+            echo "Pass: The PR has the 'no-project' label."
+            exit 0
+          fi
+
+          # Check for linked issues using GraphQL
```

Comment:
Why do this with api? I think it can be done much easier with CLI gh commands. 

something like this

```
LINKED_ISSUES=$(gh pr view ${{ github.event.pull_request.number }} \
  --repo ${{ github.repository }} \
  --json closingIssuesReferences \
  --jq '.closingIssuesReferences | length')

if [ "$LINKED_ISSUES" -gt 0 ]; then
  echo "Pass: The PR is linked to $LINKED_ISSUES issue(s)."
  exit 0
fi

```